commit 74d7a94ec8019e275564f80655ca53d8a5b8bb5c (refs/changes/16/3816/1)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-12T20:31:04-05:00 (1 year, 6 months ago)
    
    OS-6890 .WAIT doesn't work as an actual target in varpd

diff --git a/usr/src/lib/varpd/Makefile b/usr/src/lib/varpd/Makefile
index daa849572e..a0e4537ab9 100644
--- a/usr/src/lib/varpd/Makefile
+++ b/usr/src/lib/varpd/Makefile
@@ -10,10 +10,10 @@
 #
 
 #
-# Copyright 2015 Joyent, Inc.
+# Copyright 2018 Joyent, Inc.
 #
 
-SUBDIRS = libvarpd .WAIT direct files svp
+SUBDIRS = libvarpd direct files svp
 
 all :=		TARGET = all
 clean :=	TARGET = clean
@@ -26,6 +26,7 @@ lint :=		TARGET = lint
 .KEEP_STATE:
 
 all clean clobber install install_h check lint: $(SUBDIRS)
+direct files svp: libvarpd
 
 $(SUBDIRS):  FRC
 	@cd $@; pwd; $(MAKE) $(TARGET)
